map/src/util/Utils.js

```diff
@@ -102,6 +109,11 @@ export function quickNaNfix(badString) {
         .replace(/: ?NaN\b/g, nil);
 }
 
+// faster than _.cloneDeep()
```

Comment:
Incorrectly. cloneDeep() need for objects with potential circular references (layers)

Reply:
OK, but when you just need a copy of JSON object, this method is really faster than lodash's cloneDeep(), according to public available benchmark results.

Reply:
Please write when use _.cloneDeep ( about potential circular references) in this comment